app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,7 @@
         android:theme="@style/AppTheme">
         <activity
             android:name="com.example.immadisairaj.codeforces.MainActivity"
-            android:screenOrientation="portrait">
+            android:configChanges="orientation|screenSize">
```

Comment:
what is the use of this?

Reply:
That is used to manually handle configuration changes in an activity. If we want to manually handle configuration changes in the app we must declare both the "orientation" and "screenSize" values in the android:configChanges attributes.

Reply:
`screenSize` is needed?

Reply:
Yes

Reply:
[Reference](https://developer.android.com/guide/topics/resources/runtime-changes)